snippets/chainAsync.md

```diff
@@ -4,10 +4,12 @@ Chains asynchronous functions.
 
 Loop through an array of functions containing asynchronous events, calling `next` when each asynchronous event has completed.
 
+The tenerary function checks the next function exists before calling it, otherwise it will exit.
+
 ```js
 const chainAsync = fns => {
   let curr = 0;
-  const next = () => fns[curr++](next);
+  const next = () => (fns[++curr] ? fns[curr](next) : false);
```

Comment:
We are not handling bad input in most snippets, so I don't think we should make this change here. It's better to allow the method to just throw an error and let developers handle it, instead of exiting unexpectedly like that.

Reply:
I don't think the changes make it exit unexpectedly, only when it reaches the end of the list, as is the expected behaviour.